toolchains/workspace-pnpm/macros.bzl

```diff
@@ -176,6 +180,10 @@ build_node_modules = rule(
             default = False,
             doc = "Only install production dependencies"
         ),
+        "local_packages": attrs.string(
+            default = "lib",
```

Comment:
@bodymindarts to accommodate locally vendored npm packages I changed the `build_node_modules.py` code to also copy those vendored files during the turbo install process.

I know we had originally decided that we want to use the top-level `lib` folder for common configs across sub-projects. Is this where we also want to store vendored npm packages, or is there another folder that we'd prefer?

Reply:
if `lib` it is a folder for `common configs` then is not a good name, also, `tracing-rs` does not look like a common config but a library

Reply:
`lib` is a folder for shared libraries....

Some shared libraries may contain common config (that is imported as a library).

Reply:
`si` uses their `lib` folder for all shared libraries so I think you're right, this is the right approach here
https://github.com/systeminit/si/tree/main/lib